pkgs/development/python-modules/terminaltexteffects/default.nix

```diff
@@ -14,14 +14,17 @@ buildPythonPackage rec {
 
   disabled = pythonOlder "3.8";
 
-   # no tests on pypi, no tags on github
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-FDDLc7oAOgSpaxmuS6Wzej+vEOuSx3WT2ntpfoET3LI=";
+  src = fetchFromGitHub {
+    owner = "ChrisBuilds";
+    repo = "terminaltexteffects";
+    rev = "refs/tags/release-${version}";
+    hash = "sha256-4stpAFCNgE5gWBkL4Unpai2Lq7hnQPZSshy5vo7AU1E=";
   };
 
   build-system = [ poetry-core ];
 
+  doCheck = false;
```

Comment:
Why `doCheck = false;` instead of `nativeCheckInputs = [ pytestCheckHook ];`?

Reply:
Because TTE is written in pure python.The tests are just testing animation and could be skipped.(Sorry for my carelessness, I did not notice that the master branch has already been updated to 0.11.🙇)